app/views/admin/terms_of_service/generates/show.html.haml

```diff
@@ -0,0 +1,41 @@
+- content_for :page_title do
+  = t('admin.terms_of_service.generates.title')
+
+- content_for :heading_actions do
+  .back-link
+    = link_to admin_terms_of_service_index_path do
+      = material_symbol 'chevron_left'
+      = t('admin.terms_of_service.back')
+
+%p.lead= t('admin.terms_of_service.generates.explanation_html')
+
+%p.lead= t('admin.terms_of_service.generates.chance_to_review_html')
+
+%hr.spacer/
+
+= simple_form_for @generator, url: admin_terms_of_service_generate_path, method: :post do |form|
+  = render 'shared/error_messages', object: @generator
+
+  .fields-group
```

Comment:
In this one and the other form/edit views ... pull out the input fields to a `_form` partial? (and then `render form` from this view)?

Reply:
This would make sense to me if multiple views reused the form, but this is the only occurence, so I do not really see the benefit.

Reply:
👍

It's definitely more clearly useful in a new+edit scenario, but also imho there's a nice mental scanning/parsing benefit when updating the views.

There's also (again IMO) a nice consistency benefit to be had by just doing it everywhere. I think we currently have form partials in most but not quite all spots.